### PR TITLE
Formatting changes to platform finetune notebook for better cookbook rendering

### PR DESCRIPTION
**Pull request by kkt-cohere**
This PR makes changes to the ConvFinQA fine-tuning notebook. The changes are primarily focused on improving the clarity and readability of the notebook.

- The `execution_count` for several code cells has been updated to reflect the correct execution order.
- The model response is now formatted as a code block with a clear title, making it easier to distinguish from the surrounding text.
- The base model response has been updated to provide a more detailed explanation of the calculation process, including intermediate steps and a final answer.
- The markdown cell describing the base model response has been updated to provide a more accurate assessment of the model's performance and the benefits of fine-tuning.

### CODE REVIEW

File: notebooks/finetuning/convfinqa_finetuning_wandb.ipynb [hunk 1]
```diff
@@ -358,21 +358,21 @@
...
    "metadata": {},
    "outputs": [
     {
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "#### Model response ####\n",
-      "add(189, 68)\n",
-      "########################\n"
+      "#### Finetuned model response ####\n",
```
Comment (yecohere) [inline notebooks/finetuning/convfinqa_finetuning_wandb.ipynb:373]: I think the #s here are breaking it, seems like they're treated as markdowns, I think you need to change these

File: notebooks/finetuning/convfinqa_finetuning_wandb.ipynb [hunk 2]
```diff
@@ -388,32 +388,39 @@
...
    "metadata": {},
    "source": [
-    "The response object is described in detail [here](https://docs.cohere.com/reference/chat#response). As you can see, the finetuned model responds in the DSL as expected and matches the ground truth. This DSL response can now be consumed by any downstream application or engine that can compute the final answer. For a comparison we show the base model response to the same inputs."
+    "As you can see, the finetuned model responds in the DSL as expected and matches the ground truth. This DSL response can now be consumed by any downstream application or engine that can compute the final answer. For comparison, we show the base model response to the same inputs."
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 2,
+   "execution_count": 53,
```
Comment (yecohere) [inline notebooks/finetuning/convfinqa_finetuning_wandb.ipynb:405]: will the count show in the cookbook?

Comment (kkt-cohere) [reply notebooks/finetuning/convfinqa_finetuning_wandb.ipynb:405]: i dont think so. i dont see it anywhere [here](https://cohere-preview-af0af6d1-01cd-49f9-8031-0273a26ab8d8.docs.buildwithfern.com/page/convfinqa-finetuning-wandb). i only see code line numbers

File: notebooks/finetuning/convfinqa_finetuning_wandb.ipynb (hunk 1 above)
Comment (kkt-cohere) [reply notebooks/finetuning/convfinqa_finetuning_wandb.ipynb:373]: so the top ## end up making this a small heading which is good i think. this bottom ones were breaking it. with --- they should render as a separation line
image

Comment (yecohere) [reply notebooks/finetuning/convfinqa_finetuning_wandb.ipynb:373]: you need to show exactly as what you  print  out in the code, otherwise it's confusing.

Comment (kkt-cohere) [reply notebooks/finetuning/convfinqa_finetuning_wandb.ipynb]: updated, let me know what you think

Comment (kkt-cohere) [reply notebooks/finetuning/convfinqa_finetuning_wandb.ipynb]: [this](https://cohere-preview-ba9f43e2-2712-4cf4-95df-ad9f21448a25.docs.buildwithfern.com/page/convfinqa-finetuning-wandb) is the manually fixed preview.

if this looks good, we would not need this PR.

**Comment by kkt-cohere** [comment]
Closing this as it is not required anymore.

**Pull request closed without merging**